src/Features/Core/Portable/MoveToNamespace/AbstractMoveToNamespaceCodeAction.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.MoveToNamespace
+{
+    internal abstract partial class AbstractMoveToNamespaceCodeAction : CodeActionWithOptions
+    {
+        private readonly IMoveToNamespaceService _changeNamespaceService;
```

Comment:
Should we be consistent with the "move" terminology? I mean, I like "change", but I think we decided to go with "move", right?